### ISSUE TITLE
Help with understanding route file style vs John Papa style

### ISSUE THREAD

**Issue by iherbivore** [dustinspecker/generator-ng-poly#268]
I asked this question on SO ([Stack Overflow](http://stackoverflow.com/questions/35412874/angular-js-ui-router-module-route-helper-provider)), but no answers. The john Papa guide shows a route helper provider, but doesn't explain why. Both the way it is set with this generator and using a route helper provider seem to work. The way this generator does it seems more straightforward. I am totally new to angular, can anyone eli5 this for me?

**Comment by dustinspecker** [comment]
Sorry, I'm not familiar with the route helper provider usage. I've been out of the loop on recent Angular ideas.

**Comment by dustinspecker** [comment]
Closing as this isn't an issue with ng-poly, but please feel free to reopen if there's anything we can do to help.
